tests/e2e/tests/AggregatesSection/AggregatesHasDynamicTitle.spec.ts

```diff
@@ -45,7 +45,7 @@ test(
     await dashboard.navigateToFloodDisasterType();
     // Assertions
     await aggregates.aggregateComponentIsVisible();
-    await map.turnOffLayer({ layerName: 'Glofas stations' });
+    await map.clickLayerCheckbox({ layerName: 'Glofas stations' });
     await map.assertAggregateTitleOnHoverOverMap();
```

Comment:
Another question: is it actually the best choice to put this in the file/folder on 'verifying aggregates section', as the functionality we test is actually in the map (hovering), the consequences just happen to be in other sections. Just wondering if you have an opinion on this. (Depending on the answer, there might be more occurences of this.)

Reply:
This can be a part of wider discussion. I believe tho in this case we are testing the Aggregates dynamic title/header and the map is strictly a trigger for asserting the correct behaviour. As reference in 121 vast majority of the test happens on the "Table" but the outcome and the test object lies somewhere else eg. "PA overview".

Reply:
ok GEFN!